extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -131,7 +131,8 @@ define('ext.wikia.adEngine.adContext', [
 	function setContext(newContext) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has a complexity of 33 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)